app/main/helpers/presenters.py

```diff
@@ -18,7 +18,7 @@ def present(self, value, question_content):
             return value
 
     def _service_id(self, value):
-        if re.findall("[a-zA-Z]", value):
+        if re.findall("[a-zA-Z]", str(value)):
             return [value]
         else:
             return re.findall("....", str(value))
```

Comment:
Is this what we want? Given the string `qwertyuiop` it will return `['qwert', 'yuiop']`